src/com/lilithsthrone/game/Game.java

```diff
@@ -986,18 +979,22 @@ private static NPC loadNPC(Document doc, Element e, String className,
 				classMap.put(className, npcClass);
 				Method m = npcClass.getMethod("loadFromXML", Element.class, Document.class, CharacterImportSetting[].class);
 				loadFromXMLMethodMap.put(npcClass, m);
-				
+
 				Constructor<? extends NPC> declaredConstructor = npcClass.getDeclaredConstructor(boolean.class);
 				constructorMap.put(npcClass, declaredConstructor);
-				NPC npc = declaredConstructor.newInstance(true);
-				m.invoke(npc, e, doc, new CharacterImportSetting[] {});
-				return npc;
-			} else {
-				Constructor<? extends NPC> declaredConstructor = constructorMap.get(npcClass);
-				NPC npc = declaredConstructor.newInstance(true);
-				loadFromXMLMethodMap.get(npcClass).invoke(npc, e, doc, new CharacterImportSetting[] {});
-				return npc;
 			}
+			Constructor<? extends NPC> declaredConstructor = constructorMap.get(npcClass);
+			NPC npc = declaredConstructor.newInstance(true);
+			loadFromXMLMethodMap.get(npcClass).invoke(npc, e, doc, new CharacterImportSetting[] {});
+			return npc;
+		} catch(ClassNotFoundException CNFE) {
```

Comment:
All-caps names like `CNFE` or `NSME` are generally reserved for constants. Here, it would be more common to use `cnfe` and `nsme`, or similar.